README.md

```diff
@@ -12,9 +12,9 @@
 - webpack
 - git and github
 

```

Comment:
- [OPTIONAL] Please kindly update your `project title`, as this will make your readme file more informative and professional. Replace the placeholder text(`Project Name`) with your project title.